pytest_docker_tools/templates.py

```diff
@@ -71,6 +75,10 @@ def __init__(self, request):
     def visit_value(self, val):
         if isinstance(val, str):
             return FixtureFormatter(self.request).format(val)
+        elif isinstance(val, fixtureref):
+            return self.request.getfixturevalue(val.name)
+        elif isinstance(getfixturemarker(val), FixtureFunctionMarker):
```

Comment:
For our purposes I think its enough to treat `getfixturemarker()` as something that returns a truthy. It returns a fixture marker or None, which means `getfixturemarker(val) != None` is enough of a test? Given this is a private API we are using (still better than my initial hasattr) it would feel nicer not to add to many assumptions/restrictions to this test in case they refactor things? That's my preference, but if you have a strong feeling the other way i'd be happy to hear your thinking.